tempto-core/src/main/java/io/prestosql/tempto/query/JdbcQueryExecutor.java

```diff
@@ -92,6 +98,16 @@ public Connection getConnection()
         return connection;
     }
 
+    @Override
+    public <T> T inNewConnection(ConnectionCallback callback)
```

Comment:
`withConnection`? or `withNewConnection`?

Other than I thought we already have something like that.

Reply:
> `withConnection`? or `withNewConnection`?

JDBI uses `useConnection` (for `void` returning methods) and `inConnection` for value-bearing methods

> Other than I thought we already have something like that.

 do we?
we have `getConnection()`

Reply:
> do we?

Unfortunately no.